### ISSUE TITLE
Decoding RPNGs containing image outputs only works on system where they were encoded

### ISSUE THREAD

**Issue by alex-ketch** [stencila/encoda#315]
I think I've narrowed down the cause of seemingly random occurring case of missing `CodeChunk` output images when going from Jupyter Notebooks -> DOCX -> HTML.

I believe the issue lies with the fact that the RPNG gets encoded with the output image `contentUrl` pointing to path on the platform where it was encoded.

Can be reproduced by:
- converting the example Jupyter Notebook to DOCX on test-Hub
- download the generated DOCX
- convert the generated DOCX to HTML by uploading it to:
    - ✅ test-Hub: The code chunk is able to find the output image on the file system
    - ❌ production-Hub: The code chunk is unable to find the output image

---

We can't use the RPNG as the output because it is styled differently from the output, and there could be multiple output images.

Potentially slow/expensive solution: Use executa to regenerate the output image
Alternatively: Encode the output image _within_ the RPNG
Alternative2: Upload images to a public bucket and use fully formed URLs to reference them

**Comment by nokome** [comment]
Mmmm, tricky one to track down, well done.

I've labelled this high priority, since it of course will break collaboration, when people share files across machines.

> Alternatively: Encode the output image within the RPNG

This is definitely, the easiest option using the `bundle()` function, and I'm surprised we are not already doing that. Or maybe, I just broke it recently with this: https://github.com/stencila/encoda/commit/2d8cc07ac1d02b04445fe5592f62b92766201e0a#diff-983b405645ab59def0f83d49da9c8823L130

In any case, should be a fairly easy fix but needs some regression tests too.
